content/docs/integrations/jira-token/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Generating Jira Token
+lastUpdated: '2021-03-16T21:00:00.0Z'
+description: How to create Personal Access Token for Jira plugin.
+---
+
+## Introduction
+
+In order to use the Jira plugin, Roadie needs an API token to communicate with Jira APIs.
+
+
+These are set within backstage at the following url:
+
+```text
+https://<tenant-name>.roadie.so/secrets
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+1. Start by visiting the user profile settings page for your Jira user: `https://id.atlassian.com/manage-profile/security/api-tokens`
```

Comment:
Could we split this up into:

Step 1: Create an API token
Step 2: Base64 encode the credentials
Step 3: Store the credentials in Roadie

Using headings like this will make the sections linkable. I can imagine wanting to link someone directly to the base64 encoding section for example.

See the [sentry docs](https://roadie.io/docs/integrations/sentry/) for an example.

Reply:
Something like this @dtuite? :
![image](https://user-images.githubusercontent.com/2392775/111310367-4a350000-865d-11eb-8bae-4349d521427b.png)

Reply:
Yeah exactly. Prefix the headings with "Step" please. e.g. "Step 2: Base64 encode the credentials"

Reply:
Fixed